mkdocs.yml

```diff
@@ -43,117 +43,127 @@ nav:
+          - FEniCS:     Documentation/Applications/fenics.md
+          - Gaussian:   Documentation/Applications/guassian.md
+          - GAMS:       Documentation/Applications/gams.md
+          - Git:        Documentation/Applications/git.md
+          - Gurobi:     Documentation/Applications/gurobi.md
+          - LAMMPS:     Documentation/Applications/lammps.md
+          - MATLAB:     Documentation/Applications/matlab.md
+          - Paraview:   Documentation/Applications/paraview.md
+          - Q-chem:     Documentation/Applications/qchem.md
+          - R:          Documentation/Applications/r.md
+          - STAR-CCM+:  Documentation/Applications/starccm+.md
+          - VASP:       Documentation/Applications/vasp.md
+          - Visit:      Documentation/Applications/visit.md
+      - Development:
+         - Languages:
+             - Julia:   Documentation/development/languages/julia/index.md
+             - C++:     Documentation/development/languages/c++/index.md
+             - R:       Documentation/development/languages/r/index..md
+             - Bash:    Documentation/development/languages/bash/index.md
+             - Fortran: Documentation/development/languages/fortran/index.md
+             - Python:
+                  - Documentation/development/languages/python/index.md
+                  - Documentation/development/languages/python/nrel_hpc_python/index.md
+                  - Documentation/development/languages/python/dask/index.md
+                  - Jupyter: Documentation/development/languages/python/jupyter/index.md
+                  - Anaconda: Documentation/development/languages/python/anaconda/index.md
+         - Programming Models:
+              - development/programming-models/index.md
+              - MPI:
+                - development/programming-models/mpi/index.md
+                - Cray-MPICH: development/programming-models/mpi/cray-mpich.md
+                - OpenMPI: development/programming-models/mpi/openmpi.md
+                - Intel-MPI: development/programming-models/mpi/intelmpi.md
+              - OpenMP:
+                - development/programming-models/openmp/index.md
+              - OpenACC:
+                - development/programming-models/openacc/index.md
+              - CUDA:
+                - development/programming-models/cuda/index.md
+         - Compilers:
+              - GNU  : Documentation/development/compilers/gnu/index.md
+              - Intel: Documentation/development/compilers/intel/index.md
+              - NVHPC: Documentation/development/compilers/nvhpc/index.md
+              - AOCC : Documentation/development/compilers/aocc/index.md
+              - Cray : Documentation/development/compilers/cray/index.md
+         - Build Tools:
+              - Documentation/development/build_tools/index.md
+              - Config Make Install : Documentation/development/build_tools/config_make_install/config_make_install.md
```

Comment:
The "Acquire" and "Modules" pages are missing. These also go together in a series with the "Building Packages" page. Should this go in the Environment section or here?